sources/install/package_web.sh

```diff
@@ -59,6 +59,7 @@ function package_web() {
     install_arjun                   # HTTP Parameter Discovery
     install_nuclei                  # Vulnerability scanner - Needed for gau install
     install_gau                     # fetches known URLs from AlienVault's Open Threat Exchange, the Wayback Machine, Common Crawl, and URLScan
+    install_katana                  # Crawling and Spidering by projectdiscovery.io
```

Comment:
In order to keep a coherent architecture, can you move this line 1 row above so that the functions are in the order in which they are called ?